### PR TITLE
Validation for BZ-2132270

### PR DESCRIPTION
**Pull request by avd-sagare**
Signed-off-by: Avdhoot <asagare@redhat.com>

### CODE REVIEW

File: tests/e2e/workloads/app/test_jenkins_simulation.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def test_git_clone(self, pod):
         relevant_pod_logs = None
         func_calls = ["NodeStageVolume", "NodeGetVolumeStats"]
         error_msg = "System has not been booted with systemd"
-
+        inode_info = '"unit":2'
```
Comment (mashetty330) [inline tests/e2e/workloads/app/test_jenkins_simulation.py:57]: @avd-sagare how this test is relevent to the steps mentioned in here https://bugzilla.redhat.com/show_bug.cgi?id=2132270#c8 ?

The verification steps: 

1) Create CephFS pvc
2) Create a pod and attach the pod to a pvc 
3) Check on which nodes the pods are running

These steps are missing!!(?)

Comment (mashetty330) [inline tests/e2e/workloads/app/test_jenkins_simulation.py:57]: add bugzilla marker

Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: These steps are executing as part of setup.

Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: Cant add 2 BZ marker. added comment "Test coverage for BZ 2132270"

File: tests/e2e/workloads/app/test_jenkins_simulation.py [hunk 2]
```diff
@@ -82,4 +82,9 @@ def test_git_clone(self, pod):
         ), f"Logs should not contain the error message '{error_msg}'"
         logger.info(f"Logs did not contain the error message '{error_msg}'")
 
+        assert not (
```
[the remark this answers is not in the document]
Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:85]: The code for this was already present in script .. check line no 59 - 68.

File: tests/e2e/workloads/app/test_jenkins_simulation.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: done

Comment (mashetty330) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: also bz marker are linked to our bugzilla and tests only run if these bzs are verified so its important to add

Comment (mashetty330) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: ack. I see step 3 you're not performing properly here. you need to first identify the node where your pod is running then fetch the csi-cephfs-plugin pod instances which are running on the same node. Here you're doing it otherway round!

Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: check line number 60 -67 . Its already there.  # Get the node running this pod and #Get the csi_cephfsplugin pod running on this node. I hope I am not wrong

Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:57]: Okay.

Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:58]: Done

File: tests/e2e/workloads/app/test_jenkins_simulation.py [hunk 3]
```diff
@@ -82,4 +85,15 @@ def test_git_clone(self, pod):
...
+            inode_info in relevant_pod_logs
+        ), f"Logs should not contain the message '{inode_info}'"
+        logger.info(f"Logs did not contain the error message '{inode_info}'")
+
+        assert not (
+            kubelet_volume_stats in relevant_pod_logs
+        ), f"Logs should not contain the message '{kubelet_volume_stats}'"
+        logger.info(f"Logs did not contain the error message '{kubelet_volume_stats}'")
+
```
Comment (PrasadDesala) [inline tests/e2e/workloads/app/test_jenkins_simulation.py:98]: Please also validate that the metrics related to the inodes are no longer available. Refer to the bug description.

Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:98]: Hi @PrasadDesala ,
As per discussion with QE offline - "_**grep the same log and verify that no kubelet_volume_stats_inodes message appears there**_" which I already did on line 59

File: tests/e2e/workloads/app/test_jenkins_simulation.py [hunk 4]
```diff
@@ -41,6 +41,7 @@ class TestJenkinsSimulation(ManageTest):
     @workloads
     @polarion_id("OCS-4668")
     @bugzilla("2096395")
+    @bugzilla("2132270")
```
[the remark this answers is not in the document]
Comment (avd-sagare) [reply tests/e2e/workloads/app/test_jenkins_simulation.py:44]: Done

**Review by mashetty330 — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Pull request merged**